lib/worker/src/startup.rs

```diff
@@ -0,0 +1,42 @@
+use crate::gcs::GcsClient;
+use crate::routes;
+use actix_web::dev::Server;
+use actix_web::{web, App, HttpServer};
+use basin_evm::EVMClient;
+use sqlx::postgres::PgPool;
+use std::net::TcpListener;
+
+pub fn start_http_server<E: EVMClient + 'static + std::marker::Sync>(
+    listener: TcpListener,
+    db_pool: PgPool,
+    evm_client: E,
+    gcs_client: GcsClient,
+) -> Result<Server, std::io::Error> {
+    let db_pool = web::Data::new(db_pool);
+    let evm_client = web::Data::new(evm_client);
+    let gcs_client = web::Data::new(gcs_client);
+
+    let server = HttpServer::new(move || {
```

Comment:
here's the HTTP server endpoints we created

Reply:
did you look at using Warp? that feels like the most modern approach... you can see some example usage here: https://github.com/sanderpick/tableland-functions/blob/main/lib/worker/src/main.rs (handling post params, etc.). what was you reasoning for going with actix_web?